src/feature.flag.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Feature flags will be determined by environment variables and all environment variables in nodejs are
+ * either string or undefined. This function will ensure that 'false', '0', 'off' etc remain falsy
+ */
+export default (flag: string | undefined): boolean => {
```

Comment:
I was thinking as an improvement, we should give this function a name as when we import it we have to specify a name each time eg. isActiveFeature